CODE_OF_CONDUCT.md

```diff
@@ -25,12 +25,11 @@ This Code of Conduct applies to all spaces managed by the Turing Way project inc
 
 ### Contact for Reporting
 
-The co-lead-investigators of _The Turing Way_ - Dr. Kirstie Whitaker ([kwhitaker@turing.ac.uk](mailto:kwhitaker@turing.ac.uk)) and Dr. Malvika Sharan ([msharan@turing.ac.uk](mailto:msharan@turing.ac.uk)) - are responsible for enforcing the Code of Conduct.
+_The Turing Way_ project delivery team, listed in the [Governance Roles documentation](https://github.com/the-turing-way/the-turing-way/blob/main/GOVERNANCE_ROLES.md), are responsible for enforcing the Code of Conduct.
 
-If you prefer not to contact the project leads, you may also contact [Anna Krystalli](mailto:a.krystalli@sheffield.ac.uk), who is our ombudsperson.
+Please email [turingway@turing.ac.uk](mailto:turingway@turing.ac.uk).
```

Comment:
This might need to be updated to theturingway@gmail.com

Reply:
Not right now. Let's keep it Turing for now until we need to move.
We are also discussing wth SSI on creating email with ssi domain in case we need to move.